workflow.rst

```diff
@@ -215,6 +215,37 @@ what actions are allowed on a blog post::
 
     // See all the available transitions for the post in the current state
     $transitions = $workflow->getEnabledTransitions($post);
+    
+In a class or a controller
+--------------------------
+
+To access workflow inside a class, use dependency injection and inject the registry in the constructor. For a controller, you can directly inject it in a method::
```

Comment:
IMO you should go to a new line after the `constructor.`

Reply:
Oups, fixed, thank you.